test/init_global.carp

```diff
@@ -6,4 +6,6 @@
 
 ;; The one allocation left after 'carp_init_globals' should be 'g' itself:
 (defn main []
-  (assert (= 1l (Debug.memory-balance))))
+  (do
```

Comment:
Is there a reason you want to explicitly return 0 here?

Reply:
Same reason, not setting the return value explicitly to 0 will return non-zero from the executable and the test suite will fail.